app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.stories.tsx

```diff
@@ -136,7 +134,8 @@ export const FormRemoteSettingsPendingStory = () => (
   <FormRemoteSettingsPending
     {...{
       isLoading: false,
-      onConfirm: action("confirm"),
+      reviewUrl:
+        "https://settings-writer.prod.mozaws.net/v1/admin/#/buckets/main-workspace/collections/nimbus-mobile-experiments/records",
```

Comment:
Is this story intended to use a prod URL? Should it also use [`REVIEW_URL`](https://github.com/mozilla/experimenter/pull/5835/files#diff-430239984ea0d023eca01d487e1e14285726983f13689b031f324e529d398c27R17)?